test/Python/numpy_conversion.swift

```diff
@@ -6,10 +6,12 @@
 import Python
 import StdlibUnittest
 
+let gc = Python.import("gc")
 var NumpyConversionTests = TestSuite("NumpyConversion")
 let numpyModule = try? Python.attemptImport("numpy")
 
 NumpyConversionTests.test("array-conversion") {
+  let numRefs = Python.len(gc.get_objects())
```

Comment:
I would prefer not adding ad-hoc reference counting checks.
Please move the `func testWithLeakCheck` extension method to a shared `PythonUnittest` module, or duplicate the extension.

Reply:
I don't think these numpy checks actually do anything. numpy allocates (untraced but not zero-memory) things internally so this isn't good anyways.